gpswearos/src/main/java/com/android/gpswearos/presentation/MainActivity.kt

```diff
@@ -0,0 +1,104 @@
+/* While this template provides a good starting point for using Wear Compose, you can always
+ * take a look at https://github.com/android/wear-os-samples/tree/main/ComposeStarter and
+ * https://github.com/android/wear-os-samples/tree/main/ComposeAdvanced to find the most up to date
+ * changes to the libraries and their usages.
+ */
+
+package com.android.gpswearos.presentation
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Devices
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.wear.compose.material.Card
+import androidx.wear.compose.material.Text
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            StatusCard(SampleData.statusSample)
+        }
+    }
+}
+
+data class SatelliteStatus(val index: String, val data: String)
+
+@Composable
+fun StatusCard(satStatues: List<SatelliteStatus>) {
+    Card({} , modifier = Modifier
```

Comment:
There is an extra lambda with the braces here you can remove.

Reply:
> There is an extra lambda with the braces here you can remove.

If I remove this, the compiler will say "No value passed for parameter 'onClick'". It seems that it shouldn't be removed.

Reply:
> There is an extra lambda with the braces here you can remove.

If I remove this, the compiler will say "No value passed for parameter 'onClick'". It seems the extra lambda shouldn't be removed. Wear OS seems to be different from Phone on this.